client/localization/extract_static_strings.py

```diff
@@ -8,6 +8,9 @@
 from bs4 import BeautifulSoup
 from bs4.element import NavigableString
 
+AVAILABLE_LANGUAGES = ['en', 'pl']
```

Comment:
should we not be getting the list of languages from the JSON file we're loading into arangodb?

Reply:
We can't do that easily from the client